bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -74,113 +76,197 @@ if args.verbose:
-    indoc = ligolw_utils.load_filename(injection_file, False, contenthandler=LIGOLWContentHandler)
+    indoc = ligolw_utils.load_filename(injection_file, False,
+                                       contenthandler=LIGOLWContentHandler)
     sim_table = table.get_table(indoc, lsctables.SimInspiralTable.tableName)
-    inj_time = numpy.array(sim_table.get_column('geocent_end_time') + 1e-9 * sim_table.get_column('geocent_end_time_ns'), dtype=numpy.float64)
+    inj_time = numpy.array(sim_table.get_column('geocent_end_time')
+                           + 1e-9 * sim_table.get_column('geocent_end_time_ns'),
+                           dtype=numpy.float64)
 
     logging.info('Determined the found injections by time')
-    left = numpy.searchsorted(time[time_sorting], inj_time - args.injection_window, side='left')
-    right = numpy.searchsorted(time[time_sorting], inj_time + args.injection_window, side='right')
+    left = numpy.searchsorted(time[time_sorting],
+                              inj_time - args.injection_window, side='left')
+    right = numpy.searchsorted(time[time_sorting],
+                               inj_time + args.injection_window, side='right')
     found = numpy.where((right-left) == 1)[0]
     missed = numpy.where((right-left) == 0)[0]
     ambiguous = numpy.where((right-left) > 1)[0]
     missed = numpy.concatenate([missed, ambiguous])
-    logging.info('Found: %s, Missed: %s Ambiguous: %s' % (len(found), len(missed), len(ambiguous)))
+    logging.info('Found: %s, Missed: %s Ambiguous: %s'
+                 % (len(found), len(missed), len(ambiguous)))
 
     if len(ambiguous) > 0:
-        logging.warn('More than one coinc trigger found associated to injection') 
+        logging.warn('More than one coinc trigger found associated to injection')
         am = numpy.arange(0, len(inj_time), 1)[left[ambiguous]]
         bm = numpy.arange(0, len(inj_time), 1)[right[ambiguous]]
 
     logging.info('Removing injections outside of analyzed time')
     ki = keep_ind(inj_time, ana_start, ana_end)
     found_within_time = numpy.intersect1d(ki, found)
     missed_within_time = numpy.intersect1d(ki, missed)
-    logging.info('Found: %s, Missed: %s' % (len(found_within_time), len(missed_within_time)))
+    logging.info('Found: %s, Missed: %s'
+                 % (len(found_within_time), len(missed_within_time)))
 
     logging.info('Removing injections in vetoed time')
-    i1, v1 = veto_indices(inj_time, [args.veto_file], ifo='H1', 
-                                    segment_name=args.segment_name)
-    i2, v2 = veto_indices(inj_time, [args.veto_file], ifo='L1', 
-                                     segment_name=args.segment_name)
-    vi = numpy.concatenate([i1, i2])
+    if multi_ifo_style:
+        vi = numpy.array([])
+        for ifo in ifo_list:
+            vi = numpy.append(vi, veto_indices(inj_time, [args.veto_file],
+                                    ifo=ifo, segment_name=args.segment_name))
+    else:
```

Comment:
is this not equivalent to the new style code with ifo list ['H1', 'L1'] ?  If so then probably not worth having 2 separate blocks here. 
Main reason to separate old/new behaviour is for different data formats.

Reply:
Yes it is - I will update to have ifo_list set as ['H1','L1'] given old style files